runtime/onert/api/nnfw/src/nnfw_api_internal.cc

```diff
@@ -1695,7 +1742,7 @@ NNFW_STATUS nnfw_session::train_export_circle(const char *path)
 
       auto subg = subgs->Get(0); // Get 1st subgraph
       if (!idx.valid() || idx.value() >= subg->tensors()->size())
-        throw std::runtime_error("Trainable tensor index is out of range");
+        return;
```

Comment:
It's just a workaround for a problem reproduced even on master to enable exporting trained circle to a file (used in my test scripts to compare weights before and after training).
My assumption is that the trained graph can have more tensors because of the tensors consumed by a Loos function, but not sure how it worked before.

BTW:
```
 auto model = ::circle::GetModel(mmapfile.buf());
      if (!model)
        throw std::runtime_error("Failed to get model from circle");
      auto subgs = model->subgraphs();
      if (!subgs || subgs->size() != 1)
        throw std::runtime_error("Circle does not has valid subgraph or has multiple subgraphs");

      auto subg = subgs->Get(0); // Get 1st subgraph
```
can be move outside `iterateTrainableTensors` but I'll try to deal with it in a separate PR.